### PR TITLE
Minimalistic `tabby` metadata extractor with datalad-metalad

### PR DESCRIPTION
**Pull request by mih**
TODO essential features:

- [x] needs to set `@id` for the root document on the dataset
- [x] needs to set `@type` for the root document on the dataset
- [x] needs to yield other records from `/dscollection`

Ping: https://github.com/psychoinformatics-de/datalad-tabby/issues/2

- Closes https://github.com/psychoinformatics-de/datalad-tabby/issues/55
- Closes https://github.com/psychoinformatics-de/datalad-tabby/issues/2

### CODE REVIEW

File: datalad_tabby/conftest.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from datalad.conftest import setup_package
 
-from datalad_next.tests.fixtures import datalad_noninteractive_ui
+pytest_plugins = "datalad_next.tests.fixtures"
```
Comment (jsheunis) [inline datalad_tabby/conftest.py:3]: At first I didn't understand where the `existing_dataset` fixture was coming from but then I found out about the `pytest_plugins` syntax that I wasn't aware of before. Neat.

**Review by jsheunis — APPROVED** [review]
Looked at the code. Looks like a standard extractor implementation apart from the customizations to provide concept-id and version-id. I haven't tested this via command line with my own sample data.

One note (not really an issue): I have git v2.31 installed, and 4 tests were skipped locally with:

```
Git configuration redirect via GIT_CONFIG_GLOBAL only supported since Git v2.32
```

**Comment by mih** [comment]
Thanks for the review!

> One note (not really an issue): I have git v2.31 installed, and 4 tests were skipped locally with:
> ```
> Git configuration redirect via GIT_CONFIG_GLOBAL only supported since Git v2.32
> ```

You will find that many (most?) `datalad-next` based tests will be skipped on your system. Anything that interacts with a user/system config will not run, because Git versions prior 2.32 do not support the isolation approach that the fixtures use. Given that v2.32 was released more than two years ago, I'd suggest upgrading Git to at least v2.32.

**Comment by mih** [comment]
I will merge this. I won't stay the way it is, but too many decision will take longer than my window for working on this will be. It is tested, it does what it does.

**Pull request merged**